packages/astro/src/core/build/generate.ts

```diff
@@ -297,7 +297,10 @@ async function generatePage(
 		mod: pageModule,
 	};
 
-	const icon = pageData.route.type === 'page' ? green('▶') : magenta('λ');
+	const icon =
+		pageData.route.type === 'page' || pageData.route.type === 'redirect'
+			? green('▶')
+			: magenta('λ');
```

Comment:
```suggestion
	const icon =
		pageData.route.type === 'page'
			? green('▶')
			: pageData.route.type === 'redirect'
			? cyan('🔀')
			: magenta('λ');
```

Reply:
That more of seems like an exchange

Reply:
![Screenshot_2023-09-17-15-07-44-65_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/withastro/astro/assets/46300090/bdb9efd5-db16-47c4-8def-0ffbb8160c2d)

My mind is also in sync with this right arrow proposal by chatgpt

Reply:
![Screenshot_2023-09-17-15-07-44-65_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/withastro/astro/assets/46300090/bdb9efd5-db16-47c4-8def-0ffbb8160c2d)

My mind is also in sync with this right arrow proposal by chatgpt

Reply:
@Princesseuh @sarah11918 thoughts?

Reply:
I also asked chat gpt ahah, not sure about what's the right emoji there

Reply:
You probably don't want to use a emoji anyway because some terminals can't render them

Reply:
Awesome, so right now the PR looks good?

Reply:
Looks fine to me, we'll review it properly during the week once we're back to work

Reply:
Awesome then, resolving this. Thanks!